dbt_platform_helper/templates/environments/main.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  environments = yamldecode(file("../../../platform-config.yml"))["environments"]
+  env_config   = { for name, config in local.environments : name => merge(lookup(local.environments, "*", {}), config) }
+  args = {
+    application    = "{{ application }}"
+    services       = yamldecode(file("../../../platform-config.yml"))["extensions"]
+    dns_account_id = local.env_config["{{ environment }}"]["accounts"]["dns"]["id"]
+  }
+}
+
```

Comment:
```suggestion

# Default Alias
provider "aws" {
  region                   = "eu-west-2"
  profile                  = "{{ aws_profile }}"
  shared_credentials_files = ["~/.aws/config"]
}

# Domain Account (dev or prod)
provider "aws" {
  region                   = "eu-west-2"
  profile                  = "{{ domain_profile }}"
  alias                    = "domain"
  shared_credentials_files = ["~/.aws/config"]
}

```

I had this on my branch, based on latest demodjango main, but perhaps no longer needed

Reply:
I don't think that is needed now.

Reply:
They are no longer in the manifests in "main" in any case (alt least not on the dev and staging envs)

Reply:
@WillGibson they're in yours. Are they important?

Reply:
We don't need them, they are removed in @tony-griffin and my branch...

https://github.com/uktrade/demodjango-deploy/pull/109/files#diff-e28733e5cb8e059df9b162944dbd813cdd3f4ff1e3e7c3b63536321c667a4641